index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <!-- エンコード指定 -->
+    <meta charset="UTF-8" />
+
+    <!-- ページ設定 -->
+    <title>DIGSMILE INC.</title>
+    <meta name="keywards" content="ページのキーワード１,ページのキーワード２,ページのキーワード３" />
+    <meta name="description" content="ページの説明" />
+    <link rel="shortcut icon" href="../img/favicon.svg" />
+    <link rel="apple-touch-icon" href="../img/favicon.svg" />
+
+    <!-- レスポンシブ対応 -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- 模写コーディングのため、Basic認証をかける -->
+    <meta name="robots" content="noindex , nofollow" />
+
+    <!-- CSS読み込み -->
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body class="pg_home">
+    <!-- ヘッダー -->
+    <header class="ly_header">
+      <div class="bl_headerUtils">
+        <!-- サイトタイトル -->
+        <h1 class="bl_headerUtils_logo">
+          <a href="#"><img src="./img/site-logo.svg" alt="DIGSMILE INC." /></a>
+        </h1>
+        <!-- ハンバーガーボタン -->
+        <button class="bl_humbergerBtn">
+          <div class="bl_humbergerBtn_icon">
+            <span class="bl_humbergerBtn_bar bl_humbergerBtn_bar__top"></span>
+            <span class="bl_humbergerBtn_bar bl_humbergerBtn_bar__mid"></span>
+            <span class="bl_humbergerBtn_bar bl_humbergerBtn_bar__bottom"></span>
+          </div>
+          <p class="bl_humbergerBtn_txt">MENU</p>
+        </button>
+        <!-- ヘッダーナビゲーション -->
+        <nav class="bl_headerNav">
+          <ul class="bl_headerNav_inner">
+            <li>
+              <a class="bl_headerNav_siteLogo" href="#"><img src="./img/site-logo2.svg" alt="#" /></a>
+            </li>
+            <li><a class="bl_headerNav_link" href="#sec_about">ABOUT US</a></li>
+            <li><a class="bl_headerNav_link" href="#sec_works">WORKS</a></li>
+            <li><a class="bl_headerNav_link" href="#sec_culture">CULTURE</a></li>
+            <li><a class="bl_headerNav_link" href="#sec_topics">TOPICS</a></li>
+            <li><a class="bl_headerNav_link" href="#sec_contact">CONTACT</a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- /.bl_headerUtils -->
+    </header>
+
+    <main class="pg_home">
+      <!-- キービジュアル -->
+      <div class="bl_keyVisual">
+        <img class="bl_keyVisual_img" src="./img/kv-img@2x.jpg" width="1100" height="640" alt="" />
+        <div class="bl_keyVisual_body">
+          <p class="bl_keyVisual_smallCopy">デザインで人を笑顔にする会社<br />DIGSMILE INC.</p>
+          <p class="bl_keyVisual_bigCopy">DESIGN<br />FOR<br />SMILE.</p>
+        </div>
+        <!-- /.bl_keyVisual_body -->
+      </div>
+      <!-- /.bl_mainVisual -->
+      <article class="ly_article">
+        <!-- ABOUT US -->
+        <div class="ly_cont ly_cont__overlay">
+          <section class="bl_overlay_bg" id="sec_about">
+            <div class="bl_overlay">
```

Comment:
ブロックの命名が混乱しているように見えます。
bl_overlayが親で、bl_overlay_hogeが子という関係にしないとどこからどこまでがブロックなのかわからなくなります。
ここは
```suggestion
          <section class="bl_overlay" id="sec_about">
            <div class="bl_overlay_inner">
```
こんな風に名前を変えると整理できるかなと思います。